# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium-shim/base/metrics/histogram_macros.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium-shim/base/metrics/histogram_macros.h
--- a/security/sandbox/chromium-shim/base/metrics/histogram_macros.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium-shim/base/metrics/histogram_macros.h	Wed Jun 12 11:10:48 2019 +0100
@@ -5,14 +5,13 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // This is a dummy version of Chromium source file base/metrics/histogram_macros.h.
-// To provide to histogram macros required in base/memory/shared_memory_win.cc
-// UMA_HISTOGRAM_ENUMERATION and UMA_HISTOGRAM_SPARSE_SLOWLY.
+// To provide empty histogram macros required for compilation.
 // We don't require Chromiums histogram collection code.
 
 #ifndef BASE_METRICS_HISTOGRAM_MACROS_H_
 #define BASE_METRICS_HISTOGRAM_MACROS_H_
 
 #define UMA_HISTOGRAM_ENUMERATION(name, sample, enum_max) do { } while (0)
-#define UMA_HISTOGRAM_SPARSE_SLOWLY(name, sample) do { } while (0)
+#define SCOPED_UMA_HISTOGRAM_TIMER(name) do { } while (0)
 
 #endif  // BASE_METRICS_HISTOGRAM_MACROS_H_